client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -51,17 +51,12 @@ const Layout = ({ hostname }) => {
   return (
     <>
       <Section
-        className={styles['section']}
-        contentLayoutName="oneColumn"
+        className={styles['root']}
+        contentLayoutName="twoColumn"
+        contentClassName={styles['layout']}
```

Comment:
The extra space is for a second future panel. Two vertical panels is the purpose of the `twoColumn` layout.